builtins/amp-img.js

```diff
@@ -275,6 +340,10 @@ export class AmpImg extends BaseElement {
 
   /** @override */
   unlayoutCallback() {
+    if (AmpImg.V1()) {
```

Comment:
There's a "Interrupt retrieval of incomplete images to free network resources" optimization below. Do we want to support that in V1?

Reply:
I kind of hoped we won't need to. But would be happy to reconsider as we go forward. Do you have a recommendation?

Reply:
I added a note for now.